.gitlab-ci.yml

```diff
@@ -1,14 +1,11 @@
 include:
   - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-golang-static.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-golang-unittests-v2.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-commits-signoffs.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-license.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
+    file:
+      - '.gitlab-ci-check-golang-lint.yml'
+      - '.gitlab-ci-check-golang-unittests-v2.yml'
+      - '.gitlab-ci-check-commits-signoffs.yml'
+      - '.gitlab-ci-check-license.yml'
+      - '.gitlab-ci-github-status-updates.yml'
```

Comment:
so they are not from the mendertesting project?

Reply:
they are, but you can simply list all the files for the project instead of adding project, file pairs

Reply:
ah ok you left the first project line there. sorry I missed that.